client/components/app.jsx

```diff
@@ -8,15 +8,17 @@ export default class App extends React.Component {
     super(props);
     this.state = {
       view: { name: 'users' },
-      user: { userName: '' }
+      user: { userName: '' },
+      userId: { userId: '' }
```

Comment:
let's make userId a property on the user object above it like so: user: { userName: ' ', userId: ' ' }

Reply:
done